pkg/monitor/monitor.go

```diff
@@ -80,6 +87,61 @@ func (m *Monitor) isSameNode(k8sNodeName string, driverNode *volume.NodeInfo) bo
 	return volume.IsNodeMatch(node, driverNode)
 }
 
+func (m *Monitor) podMonitor() error {
+	fn := func(object runtime.Object) error {
```

Comment:
Can you also add a unit test for this. Right now there are no unit tests for the health monitor but we've added support to set kubeconfig and use the fake client so we should add them for all changes going forward.

Need the following tests:
1. Create a pod with volume owned by the driver in unknown state
2. Wait for 2 minutes and make sure it is not present

1. Create a pod with volume not owned by the driver in unknown state
2. Wait for 2 minutes and make sure it is still present

Reply:
Added test. For the unknown pods, don't need to wait for 2 minutes. The watch triggers right away.